conf/messages.en

```diff
@@ -301,7 +301,7 @@ talk.ideaForNow.show=Ideas for now
 talk.title=Title
 talk.title.note=Required. A short title for your talk
 talk.format=Format
-talk.format.note=Required.
+talk.format.note=Required. An "alien" keynote is a session in our large room with everybody on a non tachnical nor agile subject but with a small link with our jobs.
```

Comment:
tachnical -> technical
with everybody -> in front of all attendees.
with a small link to our jobs -> with a connection to our jobs.